### DISCUSSION TITLE
DOM inside the model-viewer tag not rendered in webxr mode

### DISCUSSION THREAD

**Discussion by tom-r-o** [google/model-viewer#1950] [Q&A]
### Description

Tried running the first demo from this page in a separate page: https://modelviewer.dev/examples/augmentedreality/
Couldn't get the DOM to appear on top of the model.

#### Repro
- Download both files below
- Run the python web server
- Surf to repro.html via an android device
- Click on "View in your space"
- The bug: the dom inside the model-viewer tag is not visible

To run the repro: run use the repro.html page, I used a local python web server to run it (links below):

Repro.html:
https://gist.github.com/tom-r-o/2d08696fcbc94e3cfe85883d31ec5fb1

WebServer.py:
https://gist.github.com/tom-r-o/7cbabb97780592b6c449e7d41c7633f2

### Version
https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js

### Browser Affected

- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
Well, you can see the example you copied working properly, right? Sounds like there's probably an issue with your web server or the way you copied it. Perhaps try remixing our Glitch from the issue template to see if you can track down your bug?
